src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,21 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
+
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
 public class Movie implements Cloneable {
```

Comment:
Implementing Cloneable and providing a custom clone method is not common practice with JPA entities. JPA entities should be managed by the persistence context, and the clone method can lead to unexpected behavior, especially when entities are part of a managed relationship. Consider removing the Cloneable interface and the clone method.